client-app/shared/cart/composables/useCart.ts

```diff
@@ -1,16 +1,30 @@
 import { computed, Ref, ref } from "vue";
-import { getMyCart, addItemToCart, changeCartItemQuantity } from "@core/api/graphql/cart";
+import {
+  getMyCart,
+  addItemToCart,
+  changeCartItemQuantity,
+  removeCartItem,
+  removeCoupon,
+  validateCoupon,
+  addCoupon,
+  changeCartComment,
+} from "@core/api/graphql/cart";
 import { CartType, LineItemType } from "@core/api/graphql/types";
 import { Logger } from "@core/utilities";
 
 const loading: Ref<boolean> = ref(true);
 const cart: Ref<CartType> = ref({ name: "" });
+const pages: Ref<number> = ref(0);
+const itemsPerPage: Ref<number> = ref(6);
 
 export default () => {
   async function loadMyCart(): Promise<CartType> {
     loading.value = true;
     try {
       cart.value = await getMyCart();
+      if (cart.value.items && cart.value.items.length > 0) {
```

Comment:
May be simplified as (cart.value.items?.length > 0)

Reply:
Not working, i'll left it as is